packages/headless/src/controllers/core/interactive-result/headless-core-interactive-result.ts

```diff
@@ -1,16 +1,10 @@
 import {configuration} from '../../../app/reducers';
 import {loadReducerError} from '../../../utils/errors';
 import {ConfigurationSection} from '../../../state/state-sections';
-import {Result} from '../../../api/search/search/result';
 import {debounce} from 'ts-debounce';
 import {CoreEngine} from '../../..';
 
 export interface InteractiveResultCoreOptions {
-  /**
-   * The query result.
-   */
-  result: Result;
```

Comment:
The `InteractiveResultCore` component didn't actually use the result, it simply called a callback which takes care of analytics. As such, I see no reason to take a result as a parameter here. This change is necessary to declare a single core interactive result inside SmartSnippet despite the changing source.